app/src/main/java/com/khanhhuy/movie_db/adapter/CategoryAdapter.java

```diff
@@ -11,14 +11,18 @@
 import com.khanhhuy.movie_db.R;
 import com.khanhhuy.movie_db.databinding.ItemListFragmentHomeBinding;
 import com.khanhhuy.movie_db.model.CategoryList;
+import com.khanhhuy.movie_db.screen.main.home.FragmentHome;
+import com.khanhhuy.movie_db.screen.main.home.HomePresenter;
```

Comment:
ERROR: Unused import: com.khanhhuy.movie_db.screen.main.home.HomePresenter. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports